docs/src/components/ReactExample/index.tsx

```diff
@@ -16,10 +16,9 @@ interface Props {
 }
 
 const StyledExampleWrapper = styled.div`
-  ${({ theme }) => `
-    box-shadow: ${theme.orbit.boxShadowRaised};
-    border-radius: ${theme.orbit.borderRadiusLarge};
-  `}
+  box-shadow: rgb(37 42 49 / 8%) 0px 4px 8px 0px, rgb(37 42 49 / 16%) 0px 8px 24px 0px;
+  border-radius: 12px;
```

Comment:
This is `borderRadiusBadge`, but I'm not sure it's as important to use that token.

Reply:
I was also thinking that, but I'd rather leave it as a coincidence, considering that it's not a badge so it might be weird to link them like that.